op-node/bindings/systemconfig.go

```diff
@@ -0,0 +1,1668 @@
+// Code generated - DO NOT EDIT.
```

Comment:
This whole file was just copy-pasted from the `superchain-registry/validation` module [here](https://github.com/ethereum-optimism/superchain-registry/blob/main/validation/internal/bindings/system-config.go). Is there a better way to import this code? Maybe an `op-contracts` go module?

Reply:
I do not want to maintain a bindings package personally in the monorepo, it creates too many headaches. In this particular case, we only need to make a single call so I am not sure how much we actually need the full binding. An alternative approach is based on [this](https://github.com/ethereum-optimism/optimism/blob/f6f8cd53835adc6381c115ec77777c0aebf1ea61/packages/contracts-bedrock/snapshots/abi_loader.go), I recommend seeing how the functions in these files are used in other services

Reply:
Used the func sig and existing go-ethereum methods to make this call instead of relying on the large bindings file